src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml.cs

```diff
@@ -38,6 +38,20 @@ public IndexPageModel(ReviewManager manager, UserPreferenceCache preferenceCache
             List<string> search = null, List<string> languages=null, List<string> state =null,
             List<string> status =null, List<string> type =null, int pageNo=1, int pageSize=_defaultPageSize, string sortField=_defaultSortField)
         {
+            if (search.Count == 0 && languages.Count == 0 && state.Count == 0 && status.Count == 0 && type.Count == 0)
```

Comment:
why do need to check search count? Is it left over from previous change to remove search from sticky option?

Reply:
No, I only want to use the preference setting when it's a fresh page load. If there is any search or filter query parameters, we don't want to use any preference value as that means it's not a fresh page load.